src/xrpld/rpc/handlers/Submit.cpp

```diff
@@ -40,7 +40,7 @@ getFailHard(RPC::JsonContext const& context)
 }
 
 // {
-//   tx_json: <object>,
+//   tx_blob: <object>,
```

Comment:
Actually it can take both, please change it to something like` [tx_json|tx_blob]: <object>,`

Reply:
fixed

Reply:
Maybe you forgot to commit, I don't see it changed